frontend/javascripts/main.tsx

```diff
@@ -97,10 +93,24 @@ document.addEventListener("DOMContentLoaded", async () => {
   });
   message.config({ top: 30 });
   checkBrowserFeatures();
-  await Promise.all([loadFeatureToggles(), loadActiveUser(), loadHasOrganizations()]);
-  await Promise.all([loadOrganization()]);
   const containerElement = document.getElementById("main-container");
 
+  try {
+    await Promise.all([loadFeatureToggles(), loadActiveUser(), loadHasOrganizations()]);
+    await Promise.all([loadOrganization()]);
```

Comment:
there are now 4 function calls in this new try block. for `loadHasOrganizations`, you already did the testing. but what about the others? I'm not sure what their failure would cause on master (for example, what happens if `loadOrganization` fails). maybe the new error makes sense, but maybe it's too strict :thinking: could you test these scenarios, too?

Reply:
you are right that I did not test this enough!
If `loadOrganizations` fails, the same failure screen is rendered as when `loadHasOrganizations` fails. This is probably too strict. 
An obvious solution would be to move all other api calls outside the try and catch block. the disadvantage would be that one more single `await` is introduced where it is not needed (within the try-catch). But thats probably the trade-off that has to be made, I dont have a simple solution where is not the case :sweat_smile:

Reply:
I noticed that if `loadFeatureToggles` yields an error, a blank screen is rendered, so I included it in the try-catch.

Reply:
> Oh, I understand that this depends on the test scenario! During login, I got an `Action Not Found` toast and I thought that was better than the error screen. But if the user is logged in and suddenly the request fails, the blank screen is rendered.

could it be that you tested the "open wk & log in" scenario and I tested the "open wk while being logged in"? I also simply added `throw Error()` to the `loadOrganization` function (not sure whether that makes a difference).

Reply:
>could it be that you tested the "open wk & log in" scenario and I tested the "open wk while being logged in"?

yes, I think thats right :sweat_smile: